src/sentry/static/sentry/app/components/checkboxFancy.tsx

```diff
@@ -18,8 +25,13 @@ const getHoverStyles = p =>
     border: 2px solid ${p.checked ? p.theme.purple : p.theme.gray4};
   `;
 
-const CheckboxFancy = styled(({checked, disabled, ...props}) => (
-  <div role="checkbox" aria-disabled={disabled} aria-checked={checked} {...props}>
+const CheckboxFancy = styled(({checked, disabled, className}: Props) => (
```

Comment:
```suggestion
const CheckboxFancy = styled('div')<Props>(({checked, disabled, className}) => (
```

Reply:
Will that add an additional div element into the DOM?